test/new/db/cmd/cmd_afn

```diff
@@ -35,7 +35,7 @@ f@F:functions~483f4,48540
 f@F:*~483f4,48540
 afn mymain @ 0x08048540
 afn @ 0x08048540
-?e The main flag shouldn't be renamed, it comes from bin:
+"?e The main flag shouldn't be renamed, it comes from bin:"
```

Comment:
Once commands will start be porting to accept `(argc, argv)` instead of doing their own parsing, this should become `?e "The main flag shouldn't be renamed, it comes from bin:"`. The problem is that without any quotes, the char `'` is supposed to be used to wrap strings (e.g. `?e 'Hello    World'`).

For now, let's just use the "legacy" ways of wrapping commands.

Reply:
ok